djsixpack/models.py

```diff
@@ -1 +1,13 @@
-# No models
+from django.db import models
+
+
+class SixpackParticipant(models.Model):
+    """
+    A model that contains fields relating to a sixpack experiment participant.
+    """
```

Comment:
The """ that ends a multiline docstring should preferably be preceded by a blank line (https://github.com/waveaccounting/documentation/wiki/Coding-Standards#doc-strings).